LayoutTests/imported/w3c/web-platform-tests/html/rendering/unmapped-attributes-expected.txt

```diff
@@ -2,14 +2,14 @@
 
 PASS We should be in standards mode
 PASS Subframe should be in quirks mode
-FAIL <table hspace> should not be mapped to style marginLeft in CSS1Compat mode assert_equals: expected "0px" but got "200px"
-FAIL <table hspace> should not be mapped to style marginLeft in BackCompat mode assert_equals: expected "0px" but got "200px"
-FAIL <table hspace> should not be mapped to style marginRight in CSS1Compat mode assert_equals: expected "0px" but got "200px"
-FAIL <table hspace> should not be mapped to style marginRight in BackCompat mode assert_equals: expected "0px" but got "200px"
-FAIL <table vspace> should not be mapped to style marginTop in CSS1Compat mode assert_equals: expected "0px" but got "200px"
-FAIL <table vspace> should not be mapped to style marginTop in BackCompat mode assert_equals: expected "0px" but got "200px"
-FAIL <table vspace> should not be mapped to style marginBottom in CSS1Compat mode assert_equals: expected "0px" but got "200px"
-FAIL <table vspace> should not be mapped to style marginBottom in BackCompat mode assert_equals: expected "0px" but got "200px"
+PASS <table hspace> should not be mapped to style marginLeft in CSS1Compat mode assert_equals: expected "0px" but got "200px"
```

Comment:
Looks like you've got extra stuff at the end of each one of these lines.

Reply:
e.g. see https://ews-build.s3-us-west-2.amazonaws.com/macOS-BigSur-Release-WK2-Tests-EWS/22872081-13895/imported/w3c/web-platform-tests/html/rendering/unmapped-attributes-pretty-diff.html